### ISSUE TITLE
On Raspberry Pi, nvm missing

### ISSUE THREAD

**Issue by friedger** [AbsorbingChaos/bks-setup-miner#19]
When running the script the second time, I get this:

```
SCRIPT: NVM detected.
SCRIPT: Node.js not found, installing via NVM.
bash: line 50: nvm: command not found
```

**Comment by whoabuddy** [comment]
I ran into something similar on Ubuntu Desktop, there might be a better way to source the info, will review - thank you!

**Comment by whoabuddy** [comment]
This should be fixed in 6246ee0 - @friedger can you confirm if the issue is still happening?

**Comment by whoabuddy** [comment]
Closing this one out as it should be resolved by the linked PR. Feel free to re-open or file a new issue if there is still an error!
